lib/internal/test_runner/harness.js

```diff
@@ -225,7 +225,11 @@ function getGlobalRoot() {
 }
 
 async function startSubtest(subtest) {
-  await reportersSetup;
+  if (reportersSetup) {
+    // Only incur the overhead of awaiting the Promise once.
+    await reportersSetup;
+    reportersSetup = undefined;
+  }
```

Comment:
Do you happen to have any benchmarking for this change?
Did anything specific raise this change?

Reply:
@cjihrig I am curious as well what impact this has. is it that expensive to await the same promise multiple times?